test/index.test-d.ts

```diff
@@ -56,7 +56,14 @@ app.get("/dataTyped", (request, reply) => {
   reply.view<{ text: string; }>("/index", { text: "Sample data" });
 });
 
-app.listen(3000, (err, address) => {
+// Global layout setting and per route layout are mutually exclusive
```

Comment:
Code point of view, if you set both you'll get an exception. See https://github.com/fastify/point-of-view/blob/master/index.js#L696

Reply:
Then this can be uncommented, this file is never executed.

Reply:
It is used for this script: https://github.com/fastify/point-of-view/blob/master/package.json#L9

Reply:
create a second file then :).

Reply:
What changed:
- Two files now, one for overall tests, one for tests with a global layout
- Added some templates to properly tests the functionality, before whole page templates (with `!DOCTYPE` def. etc.) were used instead of partials